src/assets/sass/protocol/components/_split.scss

```diff
@@ -0,0 +1,467 @@
+            @include bidi(((float, right, left),));
+        }
+    }
+
+    .mzp-c-split-media {
+        @include border-box;
+        @include bidi(((float, right, left),));
+        width: 50%;
+        padding: 0 (get-theme('h-grid-lg') / 2);
+
+        .mzp-l-split-body-narrow & {
+            width: 66.66%;
+        }
+
+        .mzp-l-split-body-wide & {
+            width: 33.33%;
+        }
+
+        .mzp-l-split-reversed & {
+            @include bidi(((float, left, right),));
+        }
+    }
+
+}
+
+@media #{$mq-xl} {
+    .mzp-c-split-body {
+        padding: 0 (get-theme('h-grid-xl') / 2);
+    }
+
+    .mzp-c-split-media {
+        padding: 0 (get-theme('h-grid-xl') / 2);
+    }
+}
+
+
+/* -------------------------------------------------------------------------- */
+// side by side layout, grid based
+
+@supports( display:grid) {
+    @media #{$mq-lg} {
+        .mzp-c-split-container {
+            // grid is all start/end based we don't need bidi declarations for RTL support \o/
+            align-items: center;
+            display: grid;
+            grid-template-columns: 1fr 1fr;
+            grid-template-areas: "body media";
+            grid-column-gap: get-theme('h-grid-lg');
+
+            .mzp-l-split-body-narrow & {
+                grid-template-columns: 1fr 2fr;
+            }
+
+            .mzp-l-split-body-wide & {
+                grid-template-columns: 2fr 1fr;
+            }
+
+            .mzp-l-split-reversed & {
+                grid-template-areas: "media body";
+            }
+
+            .mzp-l-split-body-narrow.mzp-l-split-reversed & {
+                grid-template-columns: 2fr 1fr;
+            }
+
+            .mzp-l-split-body-wide.mzp-l-split-reversed & {
+                grid-template-columns: 1fr 2fr;
+            }
+        }
+
+        .mzp-c-split-body,
+        .mzp-c-split-media {
+            .mzp-c-split-container & { // extra specificitly to over-ride the styles that make the float fall-back work
+                float: none;
+                min-width: 0; //because of https://stackoverflow.com/questions/43311943/prevent-content-from-expanding-grid-items
+                padding: 0;
+                width: auto;
+            }
+        }
+
+        .mzp-c-split-body {
+            grid-area: body;
+        }
+
+        .mzp-c-split-media {
+            align-self: center;
+            grid-area: media;
+            justify-self: start;
+
+            .mzp-c-split-media-asset {
+                display: block;
+            }
+
+            .mzp-l-split-reversed & {
+                justify-self: end;
+            }
+        }
+    }
+
+    @media #{$mq-xl} {
+        .mzp-c-split-container {
+            grid-column-gap: get-theme('h-grid-xl');
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// constraints on height or width of media
+
+.mzp-c-split-media {
+    @media #{$mq-lg} {
+        @supports ( display:grid ) {
```

Comment:
I think the `@media` and `@supports` can go outside the top-level declaration, since they enclose everything inside anyway.
```
@media #{$mq-lg} {
    @supports (display:grid) {
        .mzp-c-split-media {
```